arrow-core/src/main/kotlin/arrow/core/extensions/ior.kt

```diff
@@ -42,7 +41,6 @@ import arrow.typeclasses.Traverse
 import arrow.typeclasses.hashWithSalt
 import arrow.undocumented
 
-@extension
 interface IorSemigroup<L, R> : Semigroup<Ior<L, R>> {
```

Comment:
```
@Deprecated(
  "Typeclass instance have been moved to the companion object of the typeclass.",
  ReplaceWith("Semigroup.ior()", "arrow.core.ior", "arrow.typeclasses.Semigroup"),
  DeprecationLevel.WARNING
)
```

Typeclasses that are not getting deprecated should be annoted with the above deprecation